cmd/buildah/commit.go

```diff
@@ -20,6 +21,10 @@ var (
 			Name:  "signature-policy",
 			Usage: "`pathname` of signature policy file (not usually used)",
 		},
+		cli.BoolFlag{
+			Name:  "quiet",
```

Comment:
Should we make quiet a global option? In case there are other things that we want to be quiet in the future?